config/config.go

```diff
@@ -881,6 +872,9 @@ const (
 	PackageTypeModule PackageType = "module"
 )
 
+// SupportedPackageTypes is a list of all of the valid package types.
+var SupportedPackageTypes = []PackageType{PackageTypeMlModel, PackageTypeModule}
```

Comment:
I'm mixed whether I should add slam_map and archive in this PR. On one side, there is no harm in adding it, on the other, it will require a few more lines of tests and having that in a separate PR that other people can reference when they need to add new types is nice.